src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/ApplicationEntity.kt

```diff
@@ -74,6 +74,9 @@ AND (
 AND (
     :status IS NULL OR (apa.status = :#{#status?.toString()})
 )
+AND (
+    (CAST(:apAreaId AS pg_catalog.uuid) IS NULL) OR (apa.ap_area_id = :apAreaId)
```

Comment:
I think i introduced using `AS pg_catalog.uuid` but looking elsewhere in the code `AS uuid` should be fine i think?